Sources/Turf/FeatureIdentifier.swift

```diff
@@ -1,55 +1,37 @@
 import Foundation
 
-public enum Number: Equatable {
-    case int(Int)
-    case double(Double)
+public enum FeatureIdentifier: Equatable {
+    case string(String)
+    case number(Double)
```

Comment:
We had a recent fix in mapbox-maps-ios related to this https://github.com/mapbox/mapbox-maps-ios/pull/676

Might be able to simplify our implementation in light of this change.

Reply:
Yep, mapbox/mapbox-maps-ios#715 simplifies the conversion you’re referring to.